sonic_installer/main.py

```diff
@@ -476,5 +476,15 @@ def rollback_docker(container_name):
 
     click.echo('Done')
 
+# verify the next image
+@cli.command('verify_next_image')
+def verify_reboot():
```

Comment:
Please change to:

```suggestion
@cli.command('verify-next-image')
def verify_next_image():
```

Reply:
@jleveque , do you mean to change 'verify_next_image' to 'verify-next-image'? I notice that the sub-commands are using "_", for example "binary_version".

Reply:
@jleveque , do you mean to change 'verify_next_image' to 'verify-next-image'? I notice that the sub-commands are using "_", for example "binary_version".

Reply:
Yes, currently all subcommands use underscores, but we are transitioning to using hyphens. I suggest that all new subcommands in all CLI utilities use hyphens, and we will slowly migrate all existing commands, most likely using aliases at first.

Reply:
Got it, thanks for the info, Fixed.